src/Sarif.ValidationTests/Sarif.ValidationTests.csproj

```diff
@@ -39,7 +39,11 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
-    <Reference Include="xunit.abstractions, Version=2.0.0.0, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+    <Reference Include="Sarif, Version=1.5.25.0, Culture=neutral, PublicKeyToken=a567ef0185dba9aa, processorArchitecture=MSIL">		
```

Comment:
This is disturbing. In the PR where you dropped down to v4.5.1 (PR #453), where you _removed_ the NuGet reference to the previous version of SARIF, the `PublicKeyToken` was `06c70c7db0c6cd6b`. Now it won't build unless I use `a567ef0185dba9aa`. Is it possible you built 1.5.25 with a different GeneratedKey.snk file?